src/main/java/com/artipie/rpm/misc/PackagesDiff.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/rpm-adapter/LICENSE.txt
+ */
+package com.artipie.rpm.misc;
+
+import com.google.common.collect.Maps;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Differences of packages, listed in primary and packages in the repository.
+ * @since 1.10
+ */
+public final class PackagesDiff {
+
+    /**
+     * Packages, written in primary xml.
+     * Packages file name <-> checksum map.
```

Comment:
`<` and `>` symbols looks wired in Javadoc browser (it uses simplified HTML), so it should be `&lt;-&gt;` to display `<->`